HW4/HW4.py

```diff
@@ -0,0 +1,191 @@
+import math
+
+import numpy as np
+import pandas
+import plotly.figure_factory as ff
+
+# import pydot
+import statsmodels.api
+
+# from matplotlib import pyplot as plt
+# from pandas import DataFrame
+from plotly import express as px
+from plotly import graph_objects as go
+
+# import matplotlib.pyplot as plt
+from plotly.subplots import make_subplots
+from scipy import stats
+
+# from io import StringIO
+
+# from sklearn import datasets, tree
+# from sklearn.metrics import confusion_matrix
+# from sklearn.model_selection import GridSearchCV
+# from sklearn.tree import DecisionTreeClassifier, export_graphviz
+
+# import tempfile
+# from PIL import Image
+
+data_df = pandas.read_csv("suspectheartrates.csv")
+
+
+def is_column_cat(df, column_name):
```

Comment:
I thought we need to say out loud whether the response is continuous or boolean, and whether each predictor is cat/cont. Do you mean if df[column_name].nunique() > 2, then that column is continuous?

Reply:
Yes.